cmd/swarm/config.go

```diff
@@ -208,6 +209,9 @@ func flagsOverride(currentConfig *bzzapi.Config, ctx *cli.Context) *bzzapi.Confi
 	if swapLogPath := ctx.GlobalString(SwarmSwapLogPathFlag.Name); currentConfig.SwapEnabled && swapLogPath != "" {
 		currentConfig.SwapLogPath = swapLogPath
 	}
+	if initialDepo := ctx.GlobalUint64(SwarmSwapInitialDepositFlag.Name); initialDepo != 0 {
```

Comment:
Why should we only set it if it is not equal to 0? I would say that 0 is a perfectly valid value and if the user desires to set it to zero, then we can just use this value.

Reply:
0 is the default via `DefaultInitialDepositAmount`.

So I am only setting the value if it is different than 0.

Reply:
Oke! Can be resolved for my part